galaxy.yml

```diff
@@ -2,15 +2,39 @@
 # Copyright 2023 IBM Inc. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 #
-namespace: ibm
+
+### REQUIRED
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
 name: zos_ims_operator
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+namespace: ibm
+
+# The version of the collection. Must be compatible with semantic versioning
 version: 1.1.0
+
+# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
 readme: README.md
+
+# A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+# @nicks:irc/im.site#channel'
 authors:
   - IBM Support <support@ibm.com>
+
+### OPTIONAL but strongly recommended
+# A short summary description of the collection
 description: >-
   The IBM IMS Operator Collection includes roles and playbooks used for provisioning IMS on a Wazi sandbox
+
+# The path to the license file for the collection. This path is relative to the root of the collection. This key is
+# mutually exclusive with 'license'
 license_file: LICENSE
+
+# A list of tags you want to associate with the collection for indexing/searching. A tag name has the same character
+# requirements as 'namespace' and 'name'
+# IMPORTANT: if you don't want your operator to show in the z/OS Cloud Broker import page, then remove the 'zoscb' tag
```

Comment:
Can the last comment be rephrased to something like this to explain the rationale?

"Remove `zoscb` tag if you don't wish for the operator to be publicly shown in z/OS Cloud Broker's Import page. If the collection is built with this tag and uploaded to Ansible Galaxy, it will show up in z/OS Cloud Broker's Import page for the public to use."